packages/authentication/phpleague/release-please-config.json

```diff
@@ -0,0 +1,33 @@
+{
+	"bootstrap-sha": "bf199b58f9309d68474e41cb15ef1c405042eb6e",
+	"exclude-paths": [
+		".git",
+		".idea",
+		".github",
+		".vscode",
+		"tests",
+		"scripts"
+	],
+	"release-as": "1.5.0",
+	"include-v-in-tag": true,
+	"include-component-in-tag": false,
+	"bump-minor-pre-major": true,
+	"bump-patch-for-minor-pre-major": true,
+	"packages": {
+		".": {
+			"package-name": "microsoft/kiota-authentication-phpleague",
+			"changelog-path": "CHANGELOG.md",
+			"release-type": "simple",
+			"extra-files": [
+				"src/Constants.php",
+				{
```

Comment:
I'm not sure I follow why the authentication package needs to have a reference to the abstractions one?
On the mono-repo, the main config will take care of it. On the target repo, the file does not exist.

Reply:
Thanks for calling this out. You're right, the local path would fail on the target repo. Working on https://github.com/microsoft/kiota-php/pull/13 to undo the local repo configs & only have the mono-repo root contain local repo configs